tests/unit/test_eventtype_parser.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from unittest.mock import patch
+from pytest_splunk_addon.standard_lib.addon_parser.eventtype_parser import EventTypeParser
+
+
+def test_eventtypes_can_be_parsed_and_extracted(parser):
+    assert parser.eventtypes, "eventypes can not be called or does not return any value"
```

Comment:
That Assert has to be more exact. Compare it to exact value, for now all non `0/False/None` values will assert to True